packages/sdk-js-server/src/sdk-client.ts

```diff
@@ -0,0 +1,142 @@
+import { ApiKeyStamper } from "@turnkey/api-key-stamper";
+
+import type {
+  TurnkeySDKClientConfig,
+  TurnkeySDKServerConfig,
+  TurnkeyProxyHandlerConfig,
+} from "./__types__/base";
+
+import { TurnkeySDKClientBase } from "./__generated__/sdk-client-base";
+
+import type { Request, Response, RequestHandler } from "express";
+import type {
+  NextApiRequest,
+  NextApiResponse,
+  NextApiHandler,
+} from "./__types__/base";
+
+const DEFAULT_API_PROXY_ALLOWED_METHODS = [
+  "getActivity",
+  "getApiKey",
+  "getApiKeys",
+  "getAuthenticator",
+  "getAuthenticators",
+  "getOrganization",
+  "getPolicy",
+  "getPrivateKey",
+  "getUser",
+  "getWallet",
+  "getActivities",
+  "getPolicies",
+  "listPrivateKeyTags",
+  "getPrivateKeys",
+  "getSubOrgIds",
+  "listUserTags",
+  "getUsers",
+  "getWalletAccounts",
+  "getWallets",
+  "getWhoami",
+  "createSubOrganization",
+  "emailAuth",
+  "initUserEmailRecovery",
+];
+
+export class TurnkeyServerSDK {
```

Comment:
Tiny nit: should we rename this? Feel like we could use a single naming convention (`TurnkeyServerSDK...` vs. `TurnkeySDKServer...`)